Source/Tools/LinuxEmulation/LinuxSyscalls/x32/Msg.cpp

```diff
@@ -80,17 +84,20 @@ void RegisterMsg(FEX::HLE::SyscallHandler* Handler) {
                               mq_attr* HostOld_p {};
 
                               if (newattr) {
+                                FaultSafeUserMemAccess::VerifyIsReadable(newattr, sizeof(*newattr));
                                 HostNew = *newattr;
                                 HostNew_p = &HostNew;
                               }
 
                               if (oldattr) {
+                                FaultSafeUserMemAccess::VerifyIsReadable(oldattr, sizeof(*oldattr));
```

Comment:
Does it need to be readable, though? We already check for writability below, and the variable isn't used before that.

Reply:
oop, removed